themes/shijin4/static/css/shijin4.css

```diff
@@ -194,6 +194,48 @@ nav.navbar button.btn:hover {
 	margin-top: 0px;
 }
 
+.fundraising-meter {
```

Comment:
Since these are only used on the frontpage and are not theme specific, they should go in an inline <style> tag in the partial file instead of here.